addons/project/models/project_task.py

```diff
@@ -152,7 +152,7 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
     project_id = fields.Many2one('project.project', string='Project',
         index=True, tracking=True, check_company=True, change_default=True)
     project_root_id = fields.Many2one('project.project', compute='_compute_project_root_id', search='_search_project_root_id', recursive=True)
-    task_properties = fields.Properties('Properties', definition='project_id.task_properties_definition', copy=True)
+    task_properties = fields.Properties('Properties', definition='project_id.task_properties_definition', copy=True, exportable=False)
```

Comment:
What about other Properties fields? I think the changes must be done on ORM/export level: either block exporting for all Properties or add support for the export